e2e/cypress/integration/messaging/inline_markdown_image_link_open_link_spec.js

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @messaging
+
+describe('Messaging', () => {
+    let testTeam;
+    let testChannel;
+    let testUser;
+
+    before(() => {
+        // # Login as test user and visit the newly created test channel
+        cy.apiInitSetup().then(({team, channel, user}) => {
+            testUser = user;
+            testTeam = team;
+            testChannel = channel;
+
+            cy.apiLogin(testUser);
+
+            // # Visit a test channel and post a message
+            cy.visit(`/${testTeam.name}/channels/${testChannel.name}`);
+        });
+    });
+
+    it('MM-T188 - Inline markdown image that is a link, opens the link', () => {
+        // # Enable 'Show markdown preview option in message input box' setting in Account Settings > Advanced
+        cy.get('.sidebar-header-dropdown__icon').click();
+        cy.findByText('Account Settings').should('be.visible').click();
+        cy.get('#advancedButton').should('be.visible').click();
+        cy.get('#advancedPreviewFeaturesEdit').should('be.visible').click();
+        cy.get('#advancedPreviewFeaturesmarkdown_preview').should('be.visible').check();
+        cy.get('#saveSetting').should('be.visible').click();
+        cy.reload();
```

Comment:
We can do this via API using `cy.apiSaveShowMarkdownPreviewPreference()` but since we don't have current test for such UI flow, I'm fine to do it this way. Just few comments.
- should scope elements check of advance settings within `#accountSettingsModal`
- instead of `cy.reload()` which is expensive reloading the page again, just close the account settings modal.